src/deps_hack/src/lib.rs

```diff
@@ -207,7 +207,15 @@ pub struct FluentBitSpec {
     pub labels: std::collections::BTreeMap<String, String>,
     #[serde(default)]
     pub annotations: std::collections::BTreeMap<String, String>,
+    #[serde(default, rename = "serviceAccountAnnotations")]
+    pub service_account_annotations: std::collections::BTreeMap<String, String>,
+    #[serde(default, rename = "serviceLabels")]
+    pub service_labels: Option<std::collections::BTreeMap<String, String>>,
+    #[serde(default, rename = "serviceAnnotations")]
+    pub service_annotations: Option<std::collections::BTreeMap<String, String>>,
```

Comment:
I think service_annotations and service_labels don't need to be optional.

Reply:
The reference controller is doing
```
	if len(fb.Spec.Service.Labels) != 0 {
		labels = fb.Spec.Service.Labels
	} else {
		labels = fb.Labels
	}
```
so I think you can add a `len()` method to our StringMap to implement this logic, without making service_labels optional